test/registration_spec.rb

```diff
@@ -256,20 +266,25 @@
                   version:      "12",
                   release_type: nil
                 )
-              ])
+              ], {})
 
       subject.synchronize_products([installed_sles])
     end
   end
 
   describe "#downgrade_product" do
+    before do
+      allow_any_instance_of(Registration::Registration).to receive(:connect_params)
+        .and_return(anything)
```

Comment:
Is it really possible to use `anything` as the return value? What does it actually return? To me it seems strange, I think it should be `{}` here as well...